packages/frontend/src/utils/isAuctionSettled.ts

```diff
@@ -0,0 +1,9 @@
+import { ContractState } from '@/blockchain/hooks/useAuctionState'
+
+export function isAuctionSettled(state: ContractState) {
+  return (
+    state === ContractState.AUCTION_SETTLED ||
+    state === ContractState.RAFFLE_SETTLED ||
+    state === ContractState.CLAIMING_CLOSED
```

Comment:
```suggestion
    [ContractState.AUCTION_SETTLED, ContractState.RAFFLE_SETTLED, ContractState.CLAIMING_CLOSED].includes(state)
```
Or maybe extract the array to a named const (`SettledStates`?)